example-app/SantokuApp/src/apps/app/navigators/RootStackNav.tsx

```diff
@@ -59,7 +59,7 @@ const useRootStackNavigator = (initialData: AppInitialData) => {
               name="DemoStackNav"
               component={DemoStackNav}
               options={{
```

Comment:
It might be better to remove `options`.

Reply:
Fixed
50a8e76c9b12e3d48fd90eb55ba4f1299b6c0a32